src/Extensions/Compatability/Logger/Categories.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+namespace Microsoft.Omex.Extensions.Compatability.Logger
+{
+	/// <summary>
```

Comment:
>summary [](start = 6, length = 7)

nit: two empty lines between properties, applies throughout the file

Reply:
Fixed, thanks